src/Experiment.jsx

```diff
@@ -89,20 +89,20 @@ class Experiment extends React.Component<ExperimentProps> {
     );
   };
 
-  storeKey = `scientist.${this.props.userId ? this.props.userId : 'guest'}.${
-    this.props.name
-  }`;
+  storeKey = `scientist.${
+    this.props.userId !== undefined ? this.props.userId : 'guest'
```

Comment:
probably don't want to create keys like `scientist.`, `scientist.null`, `scientist.false` right?